Riot/Modules/Common/Toasts/ActivityIndicatorToastView.swift

```diff
@@ -0,0 +1,83 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import UIKit
+import DesignKit
+
+class ActivityIndicatorToastView: UIView, Themable {
```

Comment:
Will it be possible to implement it in SwiftUI?

Reply:
Ah sorry, this code already exists.

Reply:
The toast itself will need to be in `UIKit` because we want it to show even on iOS < 13, but it will certainly be possible to trigger this from SwiftUI. I have not added the relevant mechanism for this yet, but I don't imagine it would be very complicated.